aineko/extras/fastapi/health.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 Aineko Authors
+# SPDX-License-Identifier: Apache-2.0
+"""Health endpoint for FastAPI server."""
+
+from fastapi import APIRouter
+
+router = APIRouter()
+
+
+@router.get("/health", status_code=200)
+async def read_health() -> dict:
```

Comment:
I don't think this needs to be `async`. We're not using `await` so it doesn't add anything.

Reply:
I think FastAPI recommends to always use `async def` unless you need to blocking operations.
 
https://fastapi.tiangolo.com/async/#path-operation-functions

Reply:
@Barabazs Thanks! good to know